frontend/client/src/components/NavBar.js

```diff
@@ -25,10 +26,18 @@ const NavBarBase = observer((props) => {
 
   const onClickMenuItem = (num) => {
     setAnchorEl(null)
-    if (num === 4) {
+    if (num === 0) {
```

Comment:
Have you considered using a switch statement here instead? Seems like the place for it.

Reply:
I just experimented w switch statements and it would look like the below.  Not sure there is much to be gained by shifting the syntax to switch statements.

```javascript
const onClickMenuItem = (num) => {
    setAnchorEl(null)
    switch(num) {
      case 0:
        history.push(ROUTES.CODE_VALIDATIONS)
        break
      case 1:
        history.push(ROUTES.MANAGE_MEMBERS)
        break
      case 2: 
        history.push(ROUTES.BRANDING)
        break
      case 3:
        history.push(ROUTES.SETTINGS)
        break
      default:
        props.store.signOut()
        break
    }
}
```

Reply:
It's just another way to express the same thing